src/app/todos/todos.component.ts

```diff
@@ -7,58 +9,62 @@ import { Component, OnInit, Input } from '@angular/core';
 })
 export class TodosComponent implements OnInit {
 
-  activeFilter: string = 'all';
+  activeFilter = 'all';
 
-  todos: {
-    title: string,
-    completed: boolean,
-    edited: boolean
-  }[];
+  todos: Todo[];
+  filteredTodos: Todo[];
 
   count: number;
 
-  constructor() { }
+  service: TodosService;
+
+  constructor() {
+    this.service = new TodosService();
+  }
```

Comment:
```suggestion
  constructor(
    private readonly  todoService: TodosService;
  ) {}
```
Это тема про dependency injection, очень важная тема, сервисы надо именно так инжектить.
И старайся имена одинаковые писать, так проще по подку оиентироваться, к примеру когда ищещь места с подключением данного сервиса.